README.md

```diff
@@ -478,6 +499,107 @@ service.getDevices().then((resp) => {
   // err - exception message object or status code
 });
 ```
+<a name="Service+getDevicesEntries"></a>
+
+### service.getDevicesEntries() ⇒ <code>Promise</code>
+Sends request to get all registered devices entries.
+
+**Kind**: instance method of [<code>Service</code>](#Service)  
+**Returns**: <code>Promise</code> - Promise with a list of endpoints  
+**Example**  
+```js
+service.getDevicesEntries().then((resp) => {
+  // resp = [{"psk_id": "cHNraWQx", "uuid": "ABC"}, ...]
```

Comment:
Device UUID is now generated by the server, so all examples of UUID's in requests, responses etc. should follow the pattern `123e4567-e89b-12d3-a456-426655440000` or simmilar